### PR TITLE
Refactor to Postgres from MySQL, using Knex for connections

### PR DESCRIPTION
**Pull request by shivg**

### CODE REVIEW

File: client/utils/Utils.js [hunk 1]
```diff
@@ -4,6 +4,17 @@
 const mergeDefault = _.defaults
 
 const Utils = {
+  lowerCaseObjectKeys: function (obj) {
+    var keys = Object.keys(obj), key;
```
Comment (jacquesuys) [inline client/utils/Utils.js]: Good job, any particular preference here for var over let?

File: db/batchFetchIndeedJobsFromAPI.js [hunk 2]
```diff
@@ -59,14 +59,25 @@ var fetchAndStoreIndeedJobs = function (index, end) {
     })
     
 }
+var lowerCaseObjectKeys = function (obj) {
+  var keys = Object.keys(obj), key;
+  var l = keys.length;
```
Comment (jacquesuys) [inline db/batchFetchIndeedJobsFromAPI.js]: Same as above

File: db/batchFetchIndeedJobsFromAPI.js [hunk 3]
```diff
@@ -39,8 +39,8 @@ var fetchAndStoreIndeedJobs = function (index, end) {
   mergeObj(options, indeedOptions);
   return request(options)
     .then(function (res) {
-      console.log('query options are: ', options)
-      console.log('results are : ', res)
+      // console.log('query options are: ', options)
+      // console.log('results are : ', res)
       var results = translateIndeedJSONToDB(res.results);
```
Comment (jacquesuys) [inline db/batchFetchIndeedJobsFromAPI.js]: See above

File: client/utils/Utils.js (hunk 1 above)
Comment (shivg) [inline client/utils/Utils.js]: Think let should do fine here

File: db/batchFetchIndeedJobsFromAPI.js (hunk 2 above)
Comment (shivg) [inline db/batchFetchIndeedJobsFromAPI.js]: Modified and committed.

**Pull request merged**